plugins/scaffolder/src/components/MultistepJsonForm/schema.ts

```diff
@@ -39,22 +39,29 @@ function extractUiSchema(schema: JsonObject, uiSchema: JsonObject) {
     }
   }
 
-  if (!isObject(properties)) {
-    return;
-  }
+  if (isObject(properties)) {
+    for (const propName in properties) {
+      if (!properties.hasOwnProperty(propName)) {
+        continue;
+      }
 
-  for (const propName in properties) {
-    if (!properties.hasOwnProperty(propName)) {
-      continue;
+      const schemaNode = properties[propName];
+      if (!isObject(schemaNode)) {
+        continue;
+      }
+      const innerUiSchema = {};
+      uiSchema[propName] = innerUiSchema;
+      extractUiSchema(schemaNode, innerUiSchema);
     }
+  }
 
-    const schemaNode = properties[propName];
-    if (!isObject(schemaNode)) {
-      continue;
+  if (Array.isArray(anyOf)) {
```

Comment:
Looks like this totally works for `oneOf` and `allOf` as well, wanna add those while at it for completeness?

Reply:
👍  I'll add it